### PR TITLE
ATS-240 | Your income tax and NICs adjustment table

### PR DESCRIPTION
**Pull request by Synnnapse**

### CODE REVIEW

File: app/views/paye/paye_income_tax_and_nics.scala.html [hunk 1]
```diff
@@ -118,22 +118,24 @@ <h1 class="heading-xlarge">
...
-            </tr>
-        </thead>
-        <tbody>
-            @for(row <- viewModel.adjustments) {
-                <tr id="@row.label">
-                    <th scope="row">
-                        @Messages("paye.ats.adjustments." + row.label)
-                    </th>
-                    <td class="numeric">@includes.currency_amount(row.adjustmentAmount, true)</td>
+    @if(viewModel.adjustments.nonEmpty){
```
Comment (oc-ste) [inline app/views/paye/paye_income_tax_and_nics.scala.html]: Is this missing a test for the empty case?

Comment (Synnnapse) [reply app/views/paye/paye_income_tax_and_nics.scala.html]: Agreed 👍 added
